src/main/webapp/src/components/CourseOverview.vue

```diff
@@ -120,9 +126,70 @@ export default {
 };
 </script>
 
-<style scoped>
-.card .btn {
-    z-index: 2;
+<style lang="scss" scoped>
+.course-overview-title {
+    display: block;
+    font-size: $l-font;
+    font-weight: $normal;
+    margin-bottom: $l-space;
+    margin-left: $xl-space + $l-space;
+}
+.course-overview-container {
+    display: flex;
+    padding: 0 $xxl-space;
+    flex-wrap: wrap;
+}
+.course-card {
+    width: 260px;
+    padding: $m-space;
+    &:hover {
+        .edit-course-icon,
+        .delete-course-icon {
+            display: block;
+        }
+    }
+}
+.course-card-image-container {
     position: relative;
+    margin-bottom: $s-space;
+    width: 260px;
+    min-height: 120px;
+    max-height: auto;
+}
+.course-img {
+    max-width: 100%;
+    height: auto;
+}
+.edit-course-icon,
+.delete-course-icon {
+    position: absolute;
+    top: $xs-space;
+    display: none;
+    cursor: pointer;
+}
+.edit-course-icon {
+    right: $xs-space;
+}
+.delete-course-icon {
+    right: $xxl-space;
+}
+.course-card-date {
+    font-size: $s-font;
+    color: $dark-grey;
+    margin-bottom: $xs-space;
+}
+.course-card-title {
+    font-size: $m-font;
+    font-weight: $bold;
+    line-height: 1.5;
+    margin-bottom: $xs-space;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: -webkit-box;
```

Comment:
siehe Kommentare in #54

Reply:
added     word-break: break-word;
instead of clamp and (...)